app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js

```diff
@@ -46,7 +46,7 @@ define([
          */
         initialize: function () {
             this._super();
-            $(document).on('keydown', this.handleKeyDown.bind(this));
+            $('.modal-slide.adobe-stock-modal').on('keydown', this.handleKeyDown.bind(this));
```

Comment:
Can you please move this selector to defaults

Reply:
Done, thanks!